src/notation/internal/notationactioncontroller.cpp

```diff
@@ -547,7 +545,7 @@ bool NotationActionController::canReceiveAction(const ActionCode& code) const
     }
 
     if (!masterNotation->hasParts()) {
-        return code == UNDO_ACTION_CODE || code == REDO_ACTION_CODE || code == UNDO_HISTORY_CODE;
+        return code == UNDO_ACTION_CODE || code == REDO_ACTION_CODE;
```

Comment:
I would be interested to know what this if statement is doing. It causes `Redo` to be available on the Edit menu after deleting all the parts. But there is not actually anything to redo.

Reply:
It's a bug. I added a commit that explains what it probably should have been and that fixes it.